client/src/controllers/RevealController.ts

```diff
@@ -28,6 +28,7 @@ export class RevealController extends Controller<HTMLElement> {
     closed: { default: false, type: Boolean },
     peeking: { default: false, type: Boolean },
     peekTarget: { default: '', type: String },
+    store: { default: '', type: String },
```

Comment:
Let's call this `storageKey`, and instead use it as a key and not a value. After a bit of digging, I can see that this gets a bit confusing and we can add more flexibility if we make this the key.

```suggestion
    storageKey: { default: '', type: String },
```